src/main.scss

```
@@ -0,0 +1,113 @@
+body {
+  margin: 8px;
+  @font-face {
+    font-family: "Avenir";
+    src: url(./fonts/Avenir-Book.ttf);
+    font-weight: 300;
+    font-style: normal;
+  }
+  @font-face {
+    font-family: "Avenir";
+    src: url(./fonts/Avenir-Heavy.ttf);
+    font-weight: 700;
+    font-style: normal;
+  }
+}
```

```suggestion
  @font-face {
    font-family: "Avenir";
    src: url(./fonts/Avenir-Book.ttf);
    font-weight: 300;
    font-style: normal;
  }
  @font-face {
    font-family: "Avenir";
    src: url(./fonts/Avenir-Heavy.ttf);
    font-weight: 700;
    font-style: normal;
  }
```
move font face outside body selector.
body alreade has default margin 8px